src/lib/block_if.c

```diff
@@ -82,33 +83,33 @@ enum blockstat {
 #pragma clang diagnostic ignored "-Wpadded"
 struct blockif_elem {
 	TAILQ_ENTRY(blockif_elem) be_link;
-	struct blockif_req *be_req;
-	enum blockop be_op;
-	enum blockstat be_status;
-	pthread_t be_tid;
-	off_t be_block;
+	struct blockif_req  *be_req;
+	enum blockop	     be_op;
+	enum blockstat	     be_status;
+	pthread_t            be_tid;
+	off_t		     be_block;
 };
 
 struct blockif_ctxt {
-	int bc_magic;
```

Comment:
do we know why an assert is appropriate here instead of a dynamic check? All the lseeks appear to assume success.